cmd/common/image/inspect.go

```diff
@@ -127,14 +128,38 @@ func NewInspectCmd(runtime runtime.Runtime) *cobra.Command {
 			}
 			fmt.Fprint(cmd.OutOrStdout(), string(bytes))
 		case utils.OutputModeCustom:
+			// The regex matches a string that starts with a valid field name followed by a dot and another valid field name
+			re := regexp.MustCompile(`^([a-zA-Z_][a-zA-Z0-9_]*\.[a-zA-Z_][a-zA-Z0-9_]*)(.*)$`)
+
 			extraInfo, _ := cmd.PersistentFlags().GetString("extra-info")
 			if extraInfo == "" {
 				return fmt.Errorf("extra info not specified (see --extra-info)")
 			}
-			if info.ExtraInfo.Data[extraInfo] == nil {
+			matches := re.FindStringSubmatch(extraInfo)
+			if info.ExtraInfo.Data[matches[1]] == nil {
```

Comment:
FindStringSubmatch can be `nil` if extraInfo does not match the regex. Then, `matches[1]` would panic.